changelog/unreleased/9923

```diff
@@ -0,0 +1,6 @@
+Enhancement: Remove app name from connection error message
+
+We removed the app name from some connection messages.
+`No connection to ownCloud at http://..` was misleading as the server could have any other branding.
```

Comment:
Not sure about the rendering, but I could imagine ` does not work. Regular double quotes would do.

```suggestion
"No connection to ownCloud at http://..." was misleading as the server could have any other branding.
```

Reply:
It does work as we generate markdown